Environment_Engine/Convert/ICurve.cs

```diff
@@ -20,38 +20,40 @@
  * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
  */
 
-using BH.oM.Geometry;
-using BH.oM.Environment.Interface;
+using System.Collections.Generic;
 using BH.oM.Environment.Elements;
+using BH.oM.Geometry;
+using BH.Engine.Geometry;
+
+using BH.oM.Reflection.Attributes;
+using System.ComponentModel;
 
 namespace BH.Engine.Environment
 {
-    public static partial class Query
+    public static partial class Convert
     {
         /***************************************************/
         /**** Public Methods                            ****/
         /***************************************************/
 
-        public static ICurve Curve(this Panel buildingElementPanel)
-        {
-            return buildingElementPanel.PanelCurve;
-        }
-
-        public static ICurve Curve(this BuildingElement buildingElement)
+        [Description("BH.Engine.Environment.Convert.ToEdges => Returns a collection of Environment Edges from a BHoM Geomtry Polyline")]
+        [Input("curve", "A BHoM Geometry ICurve to be split into Environment Edges")]
+        [Output("edges", "A collection of Environment Edges")]
+        public static List<Edge> ToEdges(this ICurve curve)
```

Comment:
Think this has been discussed regarding multiple things and believe we need to find a better definition for it in general, but: I would see the action in here as a create method rather than a Convert. I would probably in general see taking a piece of geometry and returning an object as a create action rather than a convert in general. That is at least the way it is currently done in the Structure_Engine. To me this make more sense, but could be convinced otherwise, but think this is something we need to clarify.

@al-fisher @adecler @rwemay